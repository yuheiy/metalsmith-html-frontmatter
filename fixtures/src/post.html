<!--
layout: post.hbs
title: Post title
-->

<p>hello</p>

<!-- more -->

<p>world</p>
