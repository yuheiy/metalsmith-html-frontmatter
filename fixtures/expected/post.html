<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <title>Post title</title>
</head>

<body>
  <p>hello</p>

<!-- more -->

<p>world</p>

</body>

</html>
